File: Java/4.15.java
/*
Mad Libs are activities that have a person provide various words, which are then used to complete a short story in unexpected (and hopefully funny) ways.

Write a program that takes a string and integer as input, and outputs a sentence using those items as below. The program repeats until the input string is quit.
*/

import java.util.Scanner;

public class LabProgram {
   public static void main(String[] args) {
      /* Type your code here. */
      Scanner in = new Scanner(System.in);
        String word;
        int count;
        while (true) {
            word = in.next();
            count = in.nextInt();
            if (word.equals("quit") && count == 0)
                break;
            System.out.println("Eating " + count + " " + word + " a day keeps you happy and healthy.");
        }
    
   }
}

File: CPP/4.15.cpp
/*
Mad Libs are activities that have a person provide various words, which are then used to complete a short story in unexpected (and hopefully funny) ways.

Write a program that takes a string and integer as input, and outputs a sentence using those items as below. The program repeats until the input is quit.

Ex: If the input is:

apples 5
shoes 2
quit 0
the output is:

Eating 5 apples a day keeps you happy and healthy.
Eating 2 shoes a day keeps you happy and healthy.
*/

#include <iostream>
#include <string>
using namespace std;

int main() {

   /* Type your code here. */
   string userInput1 = "";
   int userInput2;

   while (true) {
      cin >> userInput1 >> userInput2;
      if (userInput1 == "quit") {
         break;
      }
      else {
         printf("Eating %d %s a day keeps you happy and healthy.\n", userInput2, userInput1.c_str());
      }
   }   
   return 0;
}
